x-pack/test/apm_api_integration/basic/tests/traces/top_traces.ts

```diff
@@ -19,13 +19,19 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
   describe('Top traces', () => {
     describe('when data is not loaded ', () => {
-      it('handles empty state', async () => {
+      it('handles empty state', async function () {
```

Comment:
Is this change necessary?

Reply:
no longer, left over from initial exploration. I'll remove it.